iceoryx_utils/include/iceoryx_utils/cxx/helplets.hpp

```diff
@@ -17,15 +17,14 @@
 #ifndef IOX_UTILS_CXX_HELPLETS_HPP
 #define IOX_UTILS_CXX_HELPLETS_HPP
 
-#include "iceoryx_utils/cxx/generic_raii.hpp"
-#include "iceoryx_utils/platform/platform_correction.hpp"
-
 #include <assert.h>
 #include <cstdint>
 #include <iostream>
 #include <limits>
 #include <type_traits>
 
+#include "iceoryx_utils/platform/platform_correction.hpp"
```

Comment:
Nice! Thanks, the platform correction has to be always the last include to cleanup the mess which other platforms produce.